src/store/sagas/digitalAssets/send.js

```diff
@@ -25,8 +25,8 @@ import {
 } from 'utils/address'
 
 import {
-  getPrivateKey,
-  GetPrivateKeyError,
+// getPrivateKey,
```

Comment:
Why just remove this code too?

Reply:
This is dead code. Comments will be completely removed during refactoring send asset process.
Actual code for getting of privateKey is https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/store/plugins/walletsPlugin.js#L524